ui/apps/platform/src/Components/NotifierConfiguration/NotifierConfigurationView.tsx

```diff
@@ -30,7 +30,7 @@ function NotifierConfigurationView({
         <>
             <Flex direction={{ default: 'column' }}>
                 <FlexItem>
-                    <Title headingLevel="h3">Delivery destinations</Title>
+                    <Title headingLevel="h2">Delivery destinations</Title>
```

Comment:
This might need `headingLevel` prop if `h2` in details but `h3` in **Review** step of wizard.
* https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ConfigDetails.tsx#L26
* https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/ReviewConfig.tsx#L28

Reply:
Good point. We'll set it to `h2` in the details and `h3` within the review step of the wizard 👍🏼